src/clj/rems/css/styles.clj

```diff
@@ -144,6 +144,26 @@
                    :border-color (get-theme-attribute :phase-bgcolor-completed "#ccc")
                    :color (get-theme-attribute :phase-color-completed :phase-color)}]]])
 
+(defn- generate-actions-float-menu
+  "The #actions floating menu can be too long for some screens. There is no clean solution for this in pure CSS
+  and to avoid yet another random JS-library we make the element scrollable on certain sizes.
+
+  We also do not want these styles to affect the mobile layout (i.e. more narrow than 992) where the actions
+  is at the bottom of everything and has any height available."
+  []
+  (list
+   (stylesheet/at-media {:min-width (u/px 992)
+                         :max-height (u/px 1080)}
+                        [:#actions ; make buttons here smaller on small screens
+                         [:.btn {:font-size "0.875rem" ; duplicates bootstrap style btn-sm
+                                 :padding "0.25rem 0.5rem"
+                                 :line-height 1.5
+                                 :border-radius "0.2rem"}]])
+   (stylesheet/at-media {:min-width (u/px 992)
+                         :max-height (u/px 1080)}
+                        [:#actions {:overflow-y :auto
+                                    :max-height "calc(100vh - 150px)"}])))
```

Comment:
This works fine on chrome, but on firefox the `position: sticky` is not active in any screen size: the actions pane doesn't follow the viewport. The overflow works fine though.

Reply:
nevermind, now that I'm rechecking it works fine in FF too

Reply:
Could we drop the max-height from this rule now?

Reply:
You mean the breakpoint? Yes I suppose so, it's then less easier to break with suddenly high content.